### ISSUE TITLE
`account.address_transactions(address_id)` broken

### ISSUE THREAD

**Issue by KevinColemanInc** [coinbase/coinbase-ruby#102]
When I call `account#address_transactions(address_id)` like it says to do in the do the API. I get a 

```
NoMethodError: undefined method `address_transactions' for #<Coinbase::Wallet::Account:0x00000006552188>
```
either the docs need to be updated to say:
`client.address_transactions(client.primary_account['id'], address['id'])`
or we need to add support for that method.

**Comment by artamonovdev** [comment]
Good issue.
There pull-request to add this method: https://github.com/coinbase/coinbase-ruby/pull/99

**Comment by sds** [comment]
Implemented in #99.
